filebeat/module/nginx/_meta/kibana/7/dashboard/ml-nginx-access-remote-ip-count-explorer.json

```diff
@@ -40,7 +40,7 @@
                             "enabled": true, 
                             "id": "3", 
                             "params": {
-                                "field": "nginx.access.remote_ip", 
+                                "field": "source.address", 
```

Comment:
I can't find this on in our ecs migration list. Seems like we missed this one.

Reply:
This was the main reason for https://github.com/elastic/beats/pull/11512. You'll see it when it's backported ;-)